edb/server/compiler/compiler.py

```diff
@@ -166,7 +168,8 @@ def compile_edgeql_script(
     compiler._std_schema = std_schema
     compiler._bootstrap_mode = True
 
-    units = compiler._compile(ctx=ctx, eql=eql.encode())
+    tokens = _edgeql_rust.tokenize(eql)
+    units = compiler._compile(ctx=ctx, eql_tokens=tokens)
```

Comment:
nit: just call them `tokens`, it's clear that they're EdgeQL from the context.

Reply:
Fixed